.github/workflows/build-scan-push.yml

```diff
@@ -86,9 +89,8 @@ jobs:
             docker image push ${{steps.setImageDetails.outputs.dockerRepoName}}:$TAG
           done
 
-      - name: Archive anchore report
+      - name: upload Anchore Grype SARIF report
         if: ${{ failure() && matrix.target == 'production' }}
-        uses: actions/upload-artifact@v2
+        uses: github/codeql-action/upload-sarif@v1
```

Comment:
Did you get much information displayed for a failed scan? I initially took this approach based on your work for the Node.js and .NET core images, but the Code Scanning alert wouldn't give me any more information than the number of alerts, i.e. https://github.com/DEFRA/defra-docker-node/security/code-scanning

Reply:
I've popped this back to upload-artifact.